__tests__/app.test.js

```diff
@@ -248,10 +248,86 @@ describe("/api/articles/:article_id/comments", () => {
       })
   })
   describe("POST", () => {
+
     test("Status 201 - returns the posted comment", () => {
+      const incomingComment = {
+        username: "icellusedkars",
+        body: "Posting my first comment!"
+    }
+      return request(app)
+      .post('/api/articles/2/comments')
+      .send(incomingComment)
+      .expect(201)
+      .then(({ body}) => {
+          expect(body).toBeInstanceOf(Object);
+          expect(body).toHaveProperty('postedComment');
+          expect(body.postedComment).toEqual(
+            expect.objectContaining({
+              comment_id: 19,
+              article_id:2,
+              votes: 0,
+              created_at: "2020-06-20T06:24:00.000Z",
+              author: "icellusedkars",
+              body: "Posting my first comment!"
+              }))
+      })
+    })
+
+    test("Status 400 - username is not valid", () => {
+      const incomingComment = {
+        username: "Cat",
+        body: "This comment is a lie!"
+    }
+      return request(app)
+      .post('/api/articles/2/comments')
+      .send(incomingComment)
+      .expect(400)
+      .then(({ body}) => {
+        expect(body.msg).toBe('Username not found!');
```

Comment:
would be more of a 404 not found than a 400 bad request because they've technically done nothing wrong it just doesn't exist (like using an article ID of 999999)